gen3ForDevs/gen3HelmForDevelopers.md

```diff
@@ -0,0 +1,48 @@
+# Running Gen3 On A Laptop, For Devs
```

Comment:
could you add a link to this doc next to "For a full set of configuration options..." https://github.com/uc-cdis/gen3-helm#deployment-instructions?